projects/dsp-ui/src/lib/action/components/confirmation-dialog/confirmation-dialog.component.spec.ts

```diff
@@ -19,14 +21,22 @@ class ConfirmationDialogTestHostComponent {
 
     confirmationDialogResponse: string;
 
+    testValue: ReadIntValue;
+
     constructor(private dialog: MatDialog) {
     }
 
     openDialog() {
+        this.testValue = new ReadIntValue();
```

Comment:
Could you use a mocked int value from dsp-js-lib?

Reply:
fixed in d9cde94